src/test/resources/com/puppycrawl/tools/checkstyle/checks/naming/parametername/InputParameterNameLambda.java

```diff
@@ -0,0 +1,25 @@
+package com.puppycrawl.tools.checkstyle.checks.naming.parametername;
+
+import java.util.function.BiFunction;
+import java.util.function.Function;
+
+public class InputParameterNameLambda {
```

Comment:
please add multiline comment above to make it clear that no violations are expected, as lambda is covered by  another Check.

Reply:
Done